core/src/main/java/org/radargun/Operation.java

```diff
@@ -101,6 +103,14 @@ public Operation derive(String variant) {
       return operation;
    }
 
+   public long getIntendedTime() {
```

Comment:
Ouch, there should be only a single instance of this class for each `name` :-/ It works like an enum, and for local uses it should support reference-equality comparison. And yes, I haven't written this in the javadoc...

Reply:
Well, I will wait an answer https://github.com/radargun/radargun/pull/545#discussion_r191424876 before go and fix this

Reply:
What should be the best place for `intendedTime` ?

Reply:
I was looking for usage in `stats.startRequest()` and found in Stage/Logic/Stressor
What should we do in https://github.com/radargun/radargun/blob/aa57b7de437c360f5e863c65eb7a9c6b2cffa022/extensions/cache/src/main/java/org/radargun/stages/cache/background/BackgroundStressorLogic.java#L129 ?

Reply:
Just add a `startRequest(long intendedTime)` overload.